frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -645,7 +646,7 @@ interface FeatureFlagReadOnlyProps {
     excludeTitle?: boolean
 }
 
-function FeatureFlagRollout({ readOnly }: FeatureFlagReadOnlyProps): JSX.Element {
+function FeatureFlagRollout({ readOnly, isSuper }: FeatureFlagReadOnlyProps): JSX.Element {
```

Comment:
you don't have to worry about super groups, they'll never have overrides, also when they do exist, with the current code, normal overrides will stop working, so recommend removing super group handling and let `filter_groups = featureFlag.filters.groups always`